testpilot/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -28,16 +27,19 @@ export default `
                 <img class="experiment-icon" data-hook="thumbnail"></img>
               </div>
               <div class="details-sections">
-                <section data-hook="measurements-container" class="measurement">
-                  <h3 data-l10n-id="measurements">Measurements</h3>
-                  <p data-l10n-id="measurementsDisclaimer" class="disclaimer">All data is collected anonymously and used only to help us improve this test.</p>
-                  <div data-hook="measurements-html"></div>
-                  <a class="privacy-policy" data-l10n-id="experimentPrivacyNotice" data-hook="privacy-notice-url"><span data-hook="title"></span> Privacy Notice</a>
+                <section class="user-count">
+                  <span data-l10n-id="userCountContainer">There are <span data-l10n-id="userCount" class="bold" data-hook="install-count"></span>
+                  people trying <span data-hook="title"></span> right now!</span>
                 </section>
-                <section>
+                <section class="contributors-section">
                   <h3 data-l10n-id="contributorsHeading">Brought to you by</h3>
                   <ul class="contributors"></ul>
                 </section>
+                <section data-hook="measurements-container" class="measurements">
+                  <h3 data-l10n-id="measurements">Measurements &amp; Privacy</h3>
+                  <div data-hook="measurements-html" class="measurement"></div>
+                  <a class="privacy-policy" data-l10n-id="experimentPrivacyNotice" data-hook="privacy-notice-url">You can learn more about the data collection for <span data-hook="title"></span> here.</a>
+                </section>
```

Comment:
@pdehaan glad you asked. Importance is a relative term. The heaviness of the privacy stuff make the layout of the page unbalanced visually. It also sort of winds up looking like threatening legal bullshit. Putting it lower and combining it with the very clear link right next to the primary call to action makes it less in your face, while making it easily discoverable.